src/test/java/uk/gov/pay/api/it/rule/RedisDockerRule.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.pay.api.it.rule;
+
+import com.spotify.docker.client.DefaultDockerClient;
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.exceptions.DockerCertificateException;
+import com.spotify.docker.client.exceptions.DockerException;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class RedisDockerRule implements TestRule {
```

Comment:
fix all spacing issues during an assignment

Reply:
It is a shame I overlooked it. I have updated IDE code style for spaces. So should fix it for future changes.